test/health_cards/health_card_constraints_test.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'health_cards/health_card_constraints'
+
+FILEPATH_JWS_PAYLOAD = 'test/fixtures/files/example-verbose-jws-payload.json'
+FILEPATH_MINIFIED = 'test/fixtures/files/example-jws-payload-minified.json'
+
+URL_HASH = { 'urn:uuid:643e199d-1aaf-49af-8a3b-c7ae375d11ce' => 'resource:0',
+             'urn:uuid:4fe4f8d4-9b6e-4780-8ea5-6b5791230c85' => 'resource:1',
+             'urn:uuid:911791c4-5131-44ba-85bd-8e6bdf652fd4' => 'resource:2' }.freeze
+
+MOCK_JSON = { key1: 'value1', key2: 'value2' }.freeze
+
+class HealthCardConstraintsTest < ActiveSupport::TestCase
+  setup do
+    @dummy_class = Class.new
+    @dummy_class.extend(HealthCards::HealthCardConstraints)
+
+    file = File.read(FILEPATH_JWS_PAYLOAD)
+    jws_payload = JSON.parse(file)
+    bundle = jws_payload['vc']['credentialSubject']['fhirBundle']
+    @entries = bundle['entry']
+  end
+
+  test 'Bundle.entry.fullUrl elements are populated with short resource-scheme URIs (e.g., {"fullUrl": "resource:0})' do
```

Comment:
Rather than names like `this is done`, give your tests names like `some_method does this`.

Reply:
Done.